department-of-veterans-affairs/va.gov-cms
# VACMS-6812 Create audit tool for rich text fields

## Description

Closes #6812 .

## Testing done
navigated to new view and followed qa steps

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/139128845-a47de880-0b71-4513-91a4-403d535593ba.png)

## QA steps

As cms editor I can
1. navigate to /admin/content/audit/rich-text-field-audit and view a report
2. filter by content type
3. search for content within the rich text fields
4. export results to csv
- [x] Users can filter by content type
- [x] Users can see the output of long text, formatted fields in the report
- [x] Users can export the report
- [x] update url to admin/content/audit/{{name}}
- [x] update perms for content admin and admin

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
